inline_svg.gemspec

```diff
@@ -18,13 +18,9 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 2.0"
-  spec.add_development_dependency "rake"
-  spec.add_development_dependency "rspec", "~> 3.2"
-  spec.add_development_dependency "rspec_junit_formatter", "0.2.2"
-  spec.add_development_dependency "pry"
-  spec.add_development_dependency "rubocop"
+  spec.metadata['rubygems_mfa_required'] = 'true'
+  spec.required_ruby_version = '>= 3.1'
 
-  spec.add_runtime_dependency "activesupport", ">= 3.0"
-  spec.add_runtime_dependency "nokogiri", ">= 1.6"
+  spec.add_dependency "activesupport", ">= 7.0"
+  spec.add_dependency "nokogiri", ">= 1.16"
```

Comment:
the reason why I've removed `development_dependency` from here and renamed `add_runtime_dependency` to `add_dependency` is because they are known offences of [rubocop packaging](https://docs.rubocop.org/rubocop-packaging/index.html) and will be clearer with future commits when this will be merged